app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -239,6 +243,29 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                         .zoom(ZOOM_LEVEL)
                         .build();
                 mapBoxMap.setCameraPosition(cameraPosition);
+                scaleBarPlugin = new ScaleBarPlugin(mapView, mapBoxMap);
+
+                int color = isDarkTheme?R.color.bottom_bar_light:R.color.bottom_bar_dark;
+
+                final float SIZE = new TextView(getContext()).getTextSize()-1;     //Default text size for scalebar params
+                final float SCALEBAR_HEIGHT = (float) (SIZE/3.5);                  //these values are chosen to keep scalebar size comparable to other text depending upon the screen size and device resolution
+                final float SCALEBAR_BORDER_WIDTH = SIZE/10;
+                final float SCALEBAR_MARGIN = SIZE/2;
+                final int REFRESH_INTERVAL = 15;                                  //minimum refresh time in millisec
+
+                scaleBarOptions = new ScaleBarOptions(getContext())
+                    .setTextColor(color)
+                    .setTextSize(SIZE)
+                    .setBarHeight(SCALEBAR_HEIGHT)
+                    .setBorderWidth(SCALEBAR_BORDER_WIDTH)
+                    .setMetricUnit(true)
```

Comment:
It's better if we just leave whether or not to use the metric system to the default setting of scale bar. The [default](https://github.com/mapbox/mapbox-plugins-android/blob/v0.4.0-scalebar/plugin-scalebar/src/main/java/com/mapbox/pluginscalebar/ScaleBarOptions.java#L205) seems to be smart enough to identify this based on system setting. So, we can just take advantage of the default by not setting this.